src/client/components/GameCardComponent/FormComponent/FormComponent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonComponent from '../ButtonComponent/ButtonComponent';
+import './FormComponent.style.css';
+
+export default function FormComponent({ inputLabel, value }) {
+  return (
+    <form className="form">
+      <label className="form-label">{inputLabel}</label>
+      <input className="form-input" type="text" value={value} />
+      <ButtonComponent buttonTitle="Create Team" />
+    </form>
+  );
+}
```

Comment:
It is a very specific form that only got one input field and one button which means it is not extendable. It is fine but then I would maybe rename the component name to `SingleInputFormComponent`. In my mind would give a more accurate understanding of what the component is for by only looking at the name :)